lib/Alchemy/Phrasea/Core/Version.php

```diff
@@ -16,7 +16,7 @@ class Version
     /**
      * @var string
      */
-    private $number = '4.1.0-alpha.19a';
+    private $number = '4.1.0-alpha.20a';
```

Comment:
Please bump to 4.1.0-alpha.21a , because 4.1.0-alpha.20a is already use in pr #3328

Reply:
done